vue/components/ui/organisms/header/header.vue

```diff
@@ -38,7 +38,18 @@
                 <dropdown
                     v-bind:items="accountDropdownItems"
                     v-bind:visible.sync="accountDropdownVisible"
-                />
+                >
+                    <template v-slot:announcement="{ item }">
+                        <div class="announcement-dropdown-content" v-on:click="onAnnoucementsClick">

```

Comment:
Maybe `dropdown-item-announcement`, I got a little confused about what this was and had to look at the whole HTML to understand.